lib/ruby_smb/dcerpc/wkssvc.rb

```diff
@@ -23,17 +24,99 @@ module Wkssvc
       WKSTA_INFO_102 = 0x00000066
       #TODO: WKSTA_INFO_502 = 0x000001F6
 
+      # User Enum Information Level
+      WKSTA_USER_INFO_0 = 0x00000000
+      WKSTA_USER_INFO_1 = 0x00000001
+
+      # [2.2.2.1 WKSSVC_IDENTIFY_HANDLE](https://docs.microsoft.com/en-us/openspecs/windows_protocols/ms-wkst/9ef94a11-0e5c-49d7-9ac7-68d6f03565de)
+      class WkssvcIdentifyHandle < Ndr::NdrWideStringPtr; end
```

Comment:
Based on the usage I'm seeing of `server_name: "\x00"`, I'm guessing this needs to be a null-terminated string.

We should be able to just use `server_name: ''` if we change this to the null-terminated class (Stringz instead of String).

```suggestion
      class WkssvcIdentifyHandle < Ndr::NdrWideStringzPtr; end
```

That change would also help API users by ensuring they don't have to manually null-terminate the string themselves.